vnext/Microsoft.ReactNative/Modules/DeviceInfoModule.h

```diff
@@ -14,12 +15,14 @@ namespace Microsoft::ReactNative {
 // DeviceInfoHolder is created on the UI thread and stored in the PropertyBag so that the DeviceInfo module can return
 // the constants synchronously.
 struct DeviceInfoHolder {
-  DeviceInfoHolder();
+  DeviceInfoHolder(const Mso::React::IReactContext &context);
 
   static void SetCallback(
       const React::ReactPropertyBag &propertyBag,
       Mso::Functor<void(React::JSValueObject &&)> &&callback) noexcept;
-  static void InitDeviceInfoHolder(const React::ReactPropertyBag &propertyBag) noexcept;
+  static void InitDeviceInfoHolder(
+      const Mso::React::IReactContext &context,
+      const React::ReactPropertyBag &propertyBag) noexcept;
```

Comment:
>const React::ReactPropertyBag &propertyBag [](start = 6, length = 42)

There is not need to pass both: context and the property bag. The context has a property bag. Passing them both may cause unnecessary mismatches and issues.